src/mocks/login.js

```diff
@@ -1,8 +1,9 @@
 import { user } from "./data";
 
-const ISSUER = process.env["REACT_APP_OIDC_AUTHORITY_URL"];
-const SCOPE = process.env["REACT_APP_OIDC_SCOPE"];
-const CLIENT_ID = process.env["REACT_APP_OIDC_CLIENT_ID"];
+const CLIENT_URL = new URL(String(process.env["REACT_APP_CLIENT_URL"]));
+const ISSUER = new URL(String(process.env["REACT_APP_OIDC_AUTHORITY_URL"]), CLIENT_URL);
+const SCOPE = new URL(String(process.env["REACT_APP_OIDC_SCOPE"]), CLIENT_URL);
+const CLIENT_ID = new URL(String(process.env["REACT_APP_OIDC_CLIENT_ID"]), CLIENT_URL);
```

Comment:
We should consistently use dot notation instead of bracket notation.

Reply:
Done 👍